src/App.js

```diff
@@ -25,71 +63,116 @@ function App() {
       .then((res) => res.json())
       .then(
         (result) => {
-            if (result["cod"] !== '200') {
+          if (result["cod"] !== "200") {
             setIsLoaded(false);
-            if(result["cod"]=="404")
-            {
+            if (result["cod"] == "404") {
```

Comment:
Changes in code (possibly due to not fetching coordinates is setting notFound as true and  Data not found error text being printed on screen.
![image](https://user-images.githubusercontent.com/63879595/180963822-5513b8a2-1bba-4fea-b035-42c7f9c66f7f.png)

Reply:
We didn't write this code, hence I have no clarity about this one.